code-challenge/package.json

```diff
@@ -17,32 +17,31 @@
 	},
 	"homepage": "https://github.com/Retrium/dev-candidate/blob/master/code-challenge/readme.md",
 	"devDependencies": {
-		"@babel/cli": "^7.4.4",
-		"@babel/core": "^7.4.5",
-		"@babel/node": "^7.4.5",
-		"@babel/plugin-transform-runtime": "^7.4.4",
-		"@babel/preset-env": "^7.4.5",
-		"@babel/preset-react": "^7.0.0",
-		"@babel/runtime": "^7.4.5",
-		"babel-eslint": "^10.0.1",
-		"babel-loader": "^8.0.6",
-		"eslint": "^5.16.0",
-		"eslint-plugin-flowtype": "^3.9.1",
-		"eslint-plugin-import": "^2.17.3",
-		"eslint-plugin-react": "^7.13.0",
-		"eslint-plugin-react-hooks": "^1.6.0",
-		"prettier": "^1.18.2",
-		"webpack": "^4.32.2",
-		"webpack-cli": "^3.3.2",
-		"webpack-dev-middleware": "^3.7.0"
+		"@babel/cli": "7.17.6",
+		"@babel/core": "7.17.9",
+		"@babel/eslint-parser": "7.17.0",
+		"@babel/node": "7.16.8",
+		"@babel/plugin-transform-runtime": "7.17.0",
+		"@babel/preset-env": "7.16.11",
+		"@babel/preset-react": "7.16.7",
+		"@babel/runtime": "7.17.9",
+		"babel-loader": "8.2.5",
+		"eslint": "8.14.0",
+		"eslint-plugin-import": "2.26.0",
+		"eslint-plugin-react": "7.29.4",
+		"eslint-plugin-react-hooks": "4.5.0",
+		"prettier": "2.6.2",
+		"webpack": "4.46.0",
+		"webpack-cli": "4.9.2",
+		"webpack-dev-middleware": "5.3.1"
 	},
 	"dependencies": {
 		"@retrium/react-sharedb": "^0.1.0-alpha.3",
-		"@teamwork/websocket-json-stream": "^2.0.0",
-		"express": "^4.17.1",
-		"react": "^16.8.6",
-		"react-dom": "^16.8.6",
-		"sharedb": "^1.0.0-beta.23",
-		"ws": "^7.0.0"
+		"@teamwork/websocket-json-stream": "2.0.0",
+		"express": "4.18.0",
+		"react": "16.14.0",
```

Comment:
I imagined we'd upgrade React to 17, which is what we're using in the app. Any reason not to go there?

Reply:
Yes, it caused a conflict with `@retrium/react-sharedb` because that uses react 16 and the two versions didn't coexists without errors.

I would like to revisit updating the `@retrium/react-sharedb` repo, but until then, we are stuck at react 16.

Reply:
well damn. so it goes!